### PR TITLE
Updating the codeowners file

### PR DESCRIPTION
**Pull request by bbarani**
### Description
Updating the CODEOWNERS file to reflect the maintainers on MAINTAINERS.md file

### Issues Resolved
Closes: #235 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by bbarani** [comment]
CODEOWNERS file is not green yet. Couldn't find user @anshul291995. @Bukhtawar @eirsep Can you please confirm if this user needs to be added to this repo?

**Comment by Bukhtawar** [comment]
Barani,  @anshul291995 is a former contributor to this repository, we can skip access for now

**Comment by bbarani** [comment]
@Bukhtawar  @eirsep I have updated the MAINTAINERS list and removed @anshul291995 now. I notice some CI failures so need your help to remediate it before merging this PR.

File: MAINTAINERS.md
Comment (dblock) [inline MAINTAINERS.md:11]: Let's move those folks to a new Emeritus section, don't remove them from here.

**Review by dblock — APPROVED** [review]

**Review by prudhvigodithi — APPROVED** [review]

**Pull request merged**
